src/vs/workbench/browser/parts/notifications/media/notificationsToasts.css

```diff
@@ -5,7 +5,7 @@
 
 .monaco-workbench > .notifications-toasts {
 	position: absolute;
-	z-index: 1000;
+	z-index: 9999;
```

Comment:
what is the point of this change?  do with think a global change like this could have side-effects?

Reply:
This change should actually be removed, it was a temporary fix for development since the toast notifications were sitting under the setup wizard overlay they were un-clickable so I couldn't remove them while I worked. However, this shouldn't be an issue if you  can are still able to delay the toast notifications until after the user is finished with the setup wizard.